content/en/docs/setup/independent/high-availability.md

```diff
@@ -156,12 +181,11 @@ other control plane nodes. Specifically:
 - `/etc/kubernetes/pki/etcd/ca.crt`
 - `/etc/kubernetes/pki/etcd/ca.key`
 
-The admin kubeconfig will also be needed for bootstrapping the additional
-control plane nodes:
+Copy the admin kubeconfig to the other control plane nodes:
 
 - `/etc/kubernetes/admin.conf`
 
-Here is an example, but your system may differ greatly. Replace
+In the following example, replace
 `CONTROL_PLANE_IP` with the IP addresses of the other control plane nodes.
 
```

Comment:
I trimmed ruthlessly here. I don't think we need to keep explaining with every step that it's only an example.